templates/instantsearch.php

```diff
@@ -191,13 +191,17 @@ function replace_highlights_recursive (item) {
 						sortBy: ['isRefined:desc', 'count:desc', 'name:asc'],
 						limit: 10,
 					}),
-
-					/* Search powered-by widget */
-					instantsearch.widgets.poweredBy({
-						container: '#algolia-powered-by'
-					})
 				]);
 
+				if (algolia.powered_by_enabled) {
```

Comment:
@tw2113 Can you please put one space before and after `algolia.powered_by_enabled`?